application/db/model.py

```diff
@@ -203,6 +222,21 @@ class RequestMoney(Base):
     time_updated = Column(DateTime)
     time_created = Column(DateTime)
 
+class RequestMoneyHistory(Base):
+    __tablename__ = "request_money_history"
+
+    id = Column(Integer, ForeignKey('request_money.id'))
+    request = relationship('RequestMoney', uselist=False, back_populates="history")
+    amount = Column(Float, nullable=False)
```

Comment:
thought about it, added it just in case we support amount change - approve them for less amount with a memo ....